skore/pyproject.toml

```diff
@@ -117,6 +117,14 @@ convention = "numpy"
 "src/*" = ["UP007"]
 "tests/*" = ["D"]
 
+[tool.numpydoc_validation]
+checks = [
+  "all",  # Report on all checks, except the below
+  "ES01",  # No "extended summary" section
+  "EX01",  # No "Examples" section
+  "SA01",  # No "See also" section
+]
```

Comment:
We have those filters in scikit-learn: https://github.com/scikit-learn/scikit-learn/blob/fef620292973dd25ca206c8bbdff194771c857fc/sklearn/tests/test_docstrings.py#L45